docs/source/admin/quick_howto/ciab.rst

```diff
@@ -247,14 +247,19 @@ The TightVNC optional container provides a basic lightweight window manager (flu
 		:caption: CIAB Startup Using Bash Alias
 
 		# From infrastructure/cdn-in-a-box
-		alias mydc="docker-compose -f $PWD/docker-compose.yml -f $PWD/optional/docker-compose.vnc.yml"
+		alias mydc="docker-compose "` \
+			`"-f $PWD/docker-compose.yml "` \
+			`"-f $PWD/docker-compose.expose-ports.yml "` \
```

Comment:
`docker-compose.expose-ports.yml` is not necessary for VNC.

Reply:
How would you access the VNC service if the port is kept internal to the Docker network?

Reply:
oh, you mean you don't need expose the other ports. Yeah, I guess that's true.